tools/acle_template.tex

```diff
@@ -57,104 +59,115 @@
   \renewcommand{\headrulewidth}{0pt}
 }
 
-$if(linestretch)$
-  \usepackage{setspace}
-  \setstretch{$linestretch$}
-$endif$
-
-$if(chapter-formatting)$
-  \usepackage{titlesec}
-  \titleformat{\chapter}[block]
-    {\normalfont\huge\bfseries}{\thechapter.}{1em}{\Huge}
-  \titlespacing*{\chapter}{0pt}{-19pt}{0pt}
-$endif$
-
-$if(indent)$
-$else$
-  \IfFileExists{parskip.sty}{%
-    \usepackage{parskip}
-  }{% else
-    \setlength{\parindent}{0pt}
-    \setlength{\parskip}{6pt plus 2pt minus 1pt}
-  }
-$endif$
-
 $if(highlighting-macros)$
   $highlighting-macros$
 $endif$
 
-$if(numbersections)$
-  \setcounter{secnumdepth}{$if(secnumdepth)$$secnumdepth$$else$5$endif$}
-$else$
-  \setcounter{secnumdepth}{0}
-$endif$
+\usepackage{titlesec}
+\titleformat{\chapter}[block]
+  {\normalfont\huge\bfseries}{\thechapter.}{1em}{\Huge}
+\titlespacing*{\chapter}{0pt}{-19pt}{0pt}
 
-$if(pagestyle)$
-  \pagestyle{$pagestyle$}
-$endif$
+\usepackage{parskip}
+\setlength{\parindent}{0pt}
+\setlength{\parskip}{6pt plus 2pt minus 1pt}
 
-$if(author)$
-  \author{$for(author)$$author$$sep$ \and $endfor$}
-$endif$
-
-$if(date)$
-  \date{$date$}
-$endif$
+\setcounter{secnumdepth}{5}
```

Comment:
This has to do with the numbering of the sections and subsections in the body of the specs (not the TOC)? Please Worth mentioning in a comment? And also, I see that in line 153 (of the file before your changes) you have removed `\setcounter{tocdepth}{$toc-depth$}` . Instead of removing it, could you please make sure we explicitly set it with the number we want to use? And once you have these two commands, please keep them here in the same place so that we know they are dealing with the same stuff.